manifests/ldap.pp

```diff
@@ -36,6 +38,6 @@
     owner   => 'root',
     group   => 'postfix',
     content => template('postfix/postfix-ldap-aliases.cf.erb'),
-    require => Package['postfix-ldap'],
+    #TODO: Figure out if should have require package here
```

Comment:
I tried that originally but on RHEL since the package isn't there, it throws an error:

Error: Could not retrieve catalog from remote server: Error 400 on SERVER: Invalid relationship: File[/etc/postfix/ldap-aliases.cf] { before => Package[postfix-ldap] }, because Package[postfix-ldap] doesn't seem to be in the catalog